noir-projects/noir-contracts/contracts/app/nft_contract/src/types/nft_note.nr

```diff
@@ -202,12 +202,17 @@ pub struct PartialNFTNote {
 }
 
 impl PartialNFTNote {
-    pub fn commitment(self) -> Field {
-        self.commitment
+    /// Computes a partial note validity commitment. This commitment is stored in public storage when creating the
+    /// partial note in private and then is used to check that sender and note are valid upon partial note completion
+    /// in public.
+    pub fn compute_validity_commitment(self, sender: AztecAddress) -> Field {
+        let commitment_bytes: [u8; 32] = self.commitment.to_be_bytes();
+        let sender_bytes: [u8; 32] = sender.to_field().to_be_bytes();
+        // TODO(#10537): Can we use poseidon2 here given that partial_note.commitment() was computed with it? Or at
+        // least pedersen?
+        sha256_to_field(array_concat(commitment_bytes, sender_bytes))
```

Comment:
@iAmMichaelConnor using sha256 because [the issue](https://github.com/AztecProtocol/aztec-packages/issues/10537) seems to apply here.

LMK if you think it's unnecessary here (or if you think at least pedersen can be used).